src/redux/configureSTore.js

```diff
@@ -1,12 +1,15 @@
 /* eslint-disable import/no-useless-path-segments */
```

Comment:
- [ ] Let's not disable linter errors. These are here to help us write best code.